AppManager/app_manager_server.py

```diff
@@ -658,23 +657,27 @@ def copy_modified_jars(app_name):
       logging.error("Failed to create missing lib directory in: {0}.".
         format(web_inf_dir))
       return False
-
-  cp_result = subprocess.call("cp " +  appscale_home + "/AppServer_Java/" +\
-    "appengine-java-sdk-repacked/lib/user/*.jar " + lib_dir, shell=True)
-  if cp_result != 0:
-    logging.error("Failed to copy appengine-java-sdk-repacked/lib/user jars " +\
-      "to lib directory of " + app_name)
-    return False
-
-  cp_result = subprocess.call("cp " + appscale_home + "/AppServer_Java/" +\
-    "appengine-java-sdk-repacked/lib/impl/appscale-*.jar " + lib_dir, shell=True)
-
-  if cp_result != 0:
-    logging.error("Failed to copy email jars to lib directory of " + app_name)
+  try:
+    shutil_copy(appscale_home + "/AppServer_Java/" +\
+                "appengine-java-sdk-repacked/lib/user/*.jar", lib_dir)
+    shutil_copy(appscale_home + "/AppServer_Java/" +\
+                "appengine-java-sdk-repacked/lib/impl/appscale-*.jar", lib_dir)
+    shutil_copy(appscale_home + "/ext/java/*", lib_dir)
+  except IOError as io_error:
+    logging.error("Failed to copy modified jar files to lib directory of " + app_name +\
+                  " due to:" + str(io_error))
     return False
-
   return True
 
+def shutil_copy(file_path_pattern, dest):
```

Comment:
Since this differs from `shutil.copy`, maybe `pattern_copy` or something would be a more descriptive name.